contact_list.txt

```diff
@@ -0,0 +1 @@
+abhiroop abhiroop.m25902@gmail.com
```

Comment:
There won't be a contact list, we will rather have a `pariticpants.csv`. It will have the names and email ids.
Check this commit https://github.com/codeiiest-dev/Certificate-Generator/commit/18c8e2a0babc1c34e728fedf6f02c7d7963ad975
> CSV because it will be easy to export from the Excel sheet.

Reply:
got it, added changed .txt to .csv and added .csv support